webapp/www/faq/app_installation_instructions.html

```diff
@@ -0,0 +1,155 @@
+    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Catamaran:100,200,300,400,500,600,700,800,900" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Muli" rel="stylesheet">
+
+    <!-- Plugin CSS -->
+    <link rel="stylesheet" href="../device-mockups/device-mockups.min.css">
+
+    <!-- Custom styles for this template -->
+    <link href="../css/new-age.css" rel="stylesheet">
+	<link href="../css/style.css" rel="stylesheet">
+
+  </head>
+  <style>
+    .batteryTable { border-collapse:collapse; }
+    .batteryTable th { background-color: #000; color: white; border:1px solid black; text-align:center}
+    .batteryTable td, .myTable th { border:1px solid #000; text-align:center}
+    a {color:blue;}
+  </style>
+
+  <body id="page-top">
+
+    <!-- Navigation -->
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top" id="mainNav">
+      <div class="container">
+        <a class="navbar-brand js-scroll-trigger" href="https://www.canbikeco.org">Can Do Colorado eBike Program</a>
+        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+          Menu
+          <i class="fa fa-bars"></i>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+          <ul class="navbar-nav ml-auto">
+			<li class="nav-item">
+			   <a class="nav-link js-scroll-trigger" href="faq.html">FAQ</a>
+			 </li>
+            <li class="nav-item">
+			  <a class="nav-link" href="mailto:jack@bicyclecolorado.org">Contact Us!</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <section class="download bg-primary text-center" id="faq">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-10 mx-auto" style="text-align: left;">
+            <h3 class="section-heading" style="text-align: center;"><strong>CanBikeCo App Installation and Set up</strong></h3>
+			  <p></p>
+
+ <meta name="viewport" content="width=device-width, initial-scale=1">
+
+ </head>
+ <body>
+
+ <h3>Instructions</h3>
+ <p>Click on the button to open the dropdown menu. Select the video based on your phone type and follow the video guidelines.</p>
+ <p>1. Read and accept the privacy policy by clicking on the "I accept" button.</p>
+ <p>2. Please use the token given by your program administrator when "Login via token" window is prompted.</p>
+ <p>3. Select a username.</p>
+ <p>4. Complete the onboard survey.</p>
+   <nav>
+
+     <div class="dropdown">
+       <button onclick="myFunction()" class="dropbtn">Dropdown</button>
+       <div id="myDropdown" class="dropdown-content">
+         <a href="#iOS 11-12">iOS 11-12</a>
+               <div id="videoDiv">
```

Comment:
Not sure we need the dropdown. If the videos are not part of the main text, then the yellow `<body>` part ends too early, and then the header is invisible as the user scrolls. See video. I think it is fine to simplify this and replace it with a static table.